src/app/my-blog/components/blog-item/blog-item.component.ts

```diff
@@ -1,26 +1,27 @@
 import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
 import { IPost } from '@shared/interfaces/blog-item.interface';
+import { RADIUS, COLOR } from 'src/app/constants/post.constants';
 
 @Component({
   selector: 'app-blog-item',
   templateUrl: './blog-item.component.html',
-  styleUrls: ['./blog-item.component.sass'],
+  styleUrls: ['./blog-item.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class BlogItemComponent {
-  radius = 15;
-  color = '#7ba05380';
-
   @Input() public post!: IPost;
 
+  public radius = RADIUS;
+  public color = COLOR;
+
   public readonly mockedText = 'Read more...';
 
   public handleClick(post: IPost): void {
     post.toggle = !post.toggle;
     if (post.toggle) {
-      post.likeCounter++;
-    } else if (!post.toggle && post.likeCounter > 0) {
-      post.likeCounter--;
+      post.likes++;
+    } else if (!post.toggle && post.likes > 0) {
+      post.likes--;
     }
```

Comment:
overcomplicated...
Why you need to check toggle? also:
```
if (post.toggle) {
      post.likes++;
    } else if (post.likes) {
      post.likes--;
```
Also, how it possible, if toggle true (if I correct understand - like button was clicked before), so post.likes can't be less than 0, so could be:
```
post.toggle ? post.likes++ : post.likes--;
```

Reply:
Changed